pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -688,7 +681,11 @@ func (npc *NetworkPolicyController) appendRuleToPolicyChain(iptablesCmdHandler *
 		args = append(args, "-p", protocol)
 	}
 	if dPort != "" {
-		args = append(args, "--dport", dPort)
+	    if dPort == "16384" && protocol == "UDP" {
```

Comment:
Can you confirm that there are no other areas in the code that can add `--dport`?
Can it be expected that protocol will never be TCP?